pytermgui/ansi_interface.py

```diff
@@ -738,7 +741,7 @@ def translate_mouse(code: str, method: str) -> list[MouseEvent | None] | None:
             continue
 
         matches = list(pattern.finditer(sequence))
-        if matches == []:
+        if not matches:
```

Comment:
Good call on `matches == []` being a pretty odd choice, but I personally would prefer `if len(matches) == 0`, as it is much more explicit. Just saying `not matches` hides the type.

Reply:
**pylint** warns me if I don't use `not matches` for some reason

Reply:
Which version of `pylint` and `black` are you running? There seems to be some odd behaviour, and `matches == []` is accepted on my machine.

Reply:
```pylint 2.12.2
astroid 2.9.3
Python 3.10.1 (main, Dec  9 2021, 15:28:13) [Clang 12.0.8 (https://android.googlesource.com/toolchain/llvm-project c935d99d7```

Reply:
It is latest version

Reply:
I checked about `if len(matches) == 0` and it didn't yield any error about that so I will changed it to it